### PR TITLE
re-focus on tree view after opening sources

### PR DESCRIPTION
**Pull request by Eskibear**
To resolve the issue mentioned in https://github.com/microsoft/vscode/issues/158228#issuecomment-1216326912

Meanwhile refactor data structure of staticBeans/Mappings a little bit.

### CODE REVIEW

**Comment by Eskibear** [comment]
https://user-images.githubusercontent.com/2351748/184838012-2067da7a-aba6-4f34-9cfc-ca1cdd9e313e.mp4

**Review by jdneo — APPROVED** [review]

**Comment by Eskibear** [comment]
Current implmentation cannot jump to target line. Should investigate and follow https://github.com/microsoft/vscode/issues/158228#issuecomment-1216363109

**Comment by Eskibear** [comment]
Now it works well, and should pass integration tests.

https://user-images.githubusercontent.com/2351748/185070913-e4c96d27-a0d8-49b9-959a-b1020e83a016.mp4

File: src/models/symbols.ts [hunk 1]
```diff
@@ -45,10 +46,8 @@ function sanitizeFilePath(uri: string) {
     return uri.replace(/^file:\/+/, "");
 }
 
-export function navigateToLocation(symbol: { location: vscode.Location }) {
-    const {uri, range} = symbol.location ?? (symbol as any).corresponding?.location;
-    const line = range.start.line + 1; // zero-base in range.
```
Comment (Eskibear) [inline src/models/symbols.ts:50]: Previously only jump to target line, so `charater` is always "0". Now passing the accurate range as `selection`, so previous test case failed. I'll update the case.

Comment (Eskibear) [reply src/models/symbols.ts:50]: @jdneo

**Review by jdneo — APPROVED** [review]

**Pull request merged**